src/main/java/me/crespel/runtastic/RuntasticExportConverter.java

```diff
@@ -62,19 +75,31 @@ public void run(String[] args) throws Exception {
 	protected void printUsage() {
 		System.out.println("Expected arguments:");
 		System.out.println("  list <export path>");
+		System.out.println("  user <export path>");
 		System.out.println("  info <export path> <activity id>");
+		System.out.println("  photo <exort path> <photo id>");

```

Comment:
Typo: exort => export
```suggestion
		System.out.println("  photo <export path> <photo id>");
```